config/locales/pt-BR.yml

```diff
@@ -0,0 +1,174 @@
+en:
```

Comment:
This should be `br` ?

Reply:
Should be pt-BR. Sorry about that. I'll commit a fix shortly.

Reply:
Fixed. But some tests are not passing in ruby 2.3.4 (Travis CI). The issue does not seem to be related to the translation file though. Should i try to fix it?